tsconfig.base.json

```diff
@@ -0,0 +1,13 @@
+{
+  "include": ["./packages/*/src"],
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": true,
+    "baseUrl": ".",
+    "paths": {
+      "@foam/core": ["./packages/foam-core/src"],
```

Comment:
If the `@scope/package` naming convention is not mandatory, perhaps we should drop it? Alternatively we'll need another scope, since we don't own the npm user/org `@foam`. We could do `@foambubble`, but that makes it slightly confusing as folks might think it's somehow coupled to the GitHub org.